packages/react-server-dom-webpack/src/ReactFlightWebpackNodeRegister.js

```diff
@@ -15,8 +15,27 @@ const Module = require('module');
 
 module.exports = function register() {
   const CLIENT_REFERENCE = Symbol.for('react.client.reference');
+  const SERVER_REFERENCE = Symbol.for('react.server.reference');
   const PROMISE_PROTOTYPE = Promise.prototype;
 
+  // Patch bind on the server to ensure that this creates another
```

Comment:
Is this a convenience or necessity? Having bind work on them. Why only bind? What happens if you wrap and call inside instead? I guess it’s a normal function then.

Reply:
If you wrap in a closure, then that closure needs to opt back into `"use server"` so it's just like any other closure.